requirements.txt

```diff
@@ -1,2 +1,2 @@
 bokeh
-scitools-iris
+iris
```

Comment:
This is the `conda` package name, not the PyPI name... just making sure that's what you want, right?

Reply:
Yes, I think I changed it to `scitools-iris` to get the docs building but it broke the Travis checks so I changed it back and added `doc/requirements.txt` to keep a trimmed list for readthedocs integration